plugins/plugin-svn/che-plugin-svn-ext-ide/src/main/java/org/eclipse/che/plugin/svn/ide/common/SubversionOutputConsolePresenter.java

```diff
@@ -58,8 +59,7 @@ public void go(final AcceptsOneWidget container) {
     }
 
     public void print(@NotNull final String text) {
-        final String[] lines = text.split("\n");
-        for (String line : lines) {
+        for (String line : Splitter.on("\n").split(text)) {
```

Comment:
@garagatyi I make this change, because of increasing of productivity (according to some benchmarks http://demeranville.com/battle-of-the-tokenizers-delimited-text-parser-performance/ and https://dzone.com/articles/guava-splitter-vs-stringutils), which gives Splitter's split() compare to String's split(). As Guava was already used in this project, there is no additional overhead of using it here.